draft-ietf-trans-rfc6962-bis.md

```diff
@@ -235,9 +235,9 @@ community. The major changes are:
   TransItem structure, which eases extensibility and simplifies the leaf
   structure by removing one layer of abstraction.
 
-- Signatures in SCTs for X.509 Certificates no longer cover the entire
-  certificate. Instead, leaf entries for Precertificates and X.509
-  certificates include the same data.
+- Unified leaf format and SCT signatures: the structure for both certificate and
```

Comment:
Drop " and SCT signatures" ? It is not immediately obvious how the SCT signatures relate, maybe best to not mention that entirely and let the reader look up the unified leaf format definition.